pkg/conversion/meta.go

```diff
@@ -53,6 +53,12 @@ type SimpleMetaFactory struct {
 // Interpret will return the APIVersion and Kind of the JSON wire-format
 // encoding of an object, or an error.
 func (SimpleMetaFactory) Interpret(data []byte) (version, kind string, err error) {
+	var f interface{}
+	err = json.Unmarshal(data, &f)
```

Comment:
As I said in #7944, "This isn't the right place to do this. Dig into the DataVersionAndKind method call below; it already parses w/ json package. This is a hot code path and we can't add an entirely new unmarshal call."